### PR TITLE
New Article: Go Grid FAQ

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4a9164f250/how-to/). :bow:

File: content/managed-operations/gogrid-eol-faq.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+### GoGrid EOL and migration
+
+#### Will my data be accessible after the EOL date?
+
+Data will not be accessible after the EOL date. You will be responsible for migrating your data and infrastructure before the EOL.
+
+#### Who is responsible for deleting my infrastructure once I have migrated to another platform?
+
+You are responsible for managing your infrastructure up to the EOL date. Any remaining infrastructure and related services will continue to be billed per our Terms of Service up until the EOL date
```
Comment (stephamon) [inline content/managed-operations/gogrid-eol-faq.md]: missing period at the end of the sentence

File: content/managed-operations/gogrid-eol-faq.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Is it possible to extend the EOL date if I can't migrate off the platform in time?
+
+Unfortunately, it is not possible to extend the End of Life date. Please contact us to review your migration options
```
Comment (stephamon) [inline content/managed-operations/gogrid-eol-faq.md]: missing period at the end of the sentence

File: content/managed-operations/gogrid-eol-faq.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+permalink: gogrid-eol-faq/
+audit_date:
```
Comment (stephamon) [inline content/managed-operations/gogrid-eol-faq.md]: Add the audit date as today

**Comment by stephamon** [comment]
Hey @DonNateR just a few things

File: content/managed-operations/gogrid-eol-faq.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Yes, we will continue to support GoGrid and will be available to assist you with any issues you may have.
+
+#### How can Rackspace support assist with my cloud infrastructure?
+
+Cloud Servers with a Managed Operations Service Level extends the enhanced managed support level from our Managed Hosting offering into our Cloud offering. The Managed Operations Service Level provides an additional level of support that includes monitoring, OS and application infrastructure layer support, and technical guidance for Cloud Servers.
+
+#### I pre-paid for GoGrid services or I am under a contract for GoGrid services. Will I be issued a refund if I want to migrate to another platform before my term expires?
+
+For customers that have a pre-paid service or contract that extends past the End of Life (EOL) date of May 31st, 2019, they will be issued refunds for the remaining balance past the EOL date. For these customers that choose to migrate to a Rackspace product before the EOL date, they will be provided a credit towards Rackspace services equivalent to the remaining balance of their pre-paid or contractual amount calculated after those respective GoGrid services have been spun down.
```
Comment (catlook) [inline content/managed-operations/gogrid-eol-faq.md]: Customers that have a pre-paid service or contract that extends past the End of Life (EOL) date of May 31st, 2019 will be issued  ...
.... EOL date. Any of these customers who choose to migrate to a Rackspace product before the EOL date will be provided ...

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-81b2ccca82/how-to/)

**Pull request merged**